packages/in_app_purchase/in_app_purchase_android/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: in_app_purchase_android
 description: An implementation for the Android platform of the Flutter `in_app_purchase` plugin. This uses the Android BillingClient APIs.
 repository: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase/in_app_purchase_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+in_app_purchase%22
-version: 0.1.4+1
+version: 0.1.4+2
```

Comment:
If we first publish the new platform_interface we can upgrade `in_app_purchase_platform_interface` to 2.0.0 and add the property to the constructor of ProductDetails.